uint8 selected
uint8 command
uint8 auxDigit
float32[3] armDir
